packages/backend-common/api-report.md

```diff
@@ -175,7 +175,8 @@ export function createStatusCheckRouter(options: {
 // @public (undocumented)
 export class DatabaseManager {
   forPlugin(pluginId: string): PluginDatabaseManager;
-  static fromConfig(config: Config): DatabaseManager;
+  // Warning: (ae-forgotten-export) The symbol "Options" needs to be exported by the entry point index.d.ts
```

Comment:
Let's export this and call it ConfigOptions?

Reply:
Sorry @marcuseide my bad, this names does not make sense... 😅 

It should be `DatabaseManagerOptions` so it make sense when it's exported.

Reply:
Awesome 👍